roles/grafana_cloud_operator/tasks/grafana_oncall_hub_spoke.yml

```diff
@@ -102,20 +122,50 @@
       until: grafana_integration_response.status in [200, 201]
       failed_when: false
 
+    - name: Update status with managedCluster field
+      operator_sdk.util.k8s_status:
+        api_version: grafanacloud.stakater.com/v1alpha1
+        kind: Config
+        name: "{{ cr_name }}"
+        namespace: "{{ cr_namespace }}"
+        status:
+          managedClusters: []
```

Comment:
please use snake case format as used everywhere else in the project: 

managedClusters => managed_clusters